OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoLogic.cs

```diff
@@ -29,16 +29,31 @@ class GameInfoLogic : ChromeLogic
 		IngameInfoPanel activePanel;
 		readonly bool hasError;
 
+		[TranslationReference]
+		static readonly string Objectives = "objectives";
+
+		[TranslationReference]
+		static readonly string Briefing = "briefing";
+
+		[TranslationReference]
+		static readonly string Options = "options";
+
+		[TranslationReference]
+		static readonly string Debug = "debug";
+
+		[TranslationReference]
+		static readonly string Chat = "chat";
+
 		[ObjectCreator.UseCtor]
 		public GameInfoLogic(Widget widget, ModData modData, World world, IngameInfoPanel initialPanel, Action<bool> hideMenu)
 		{
 			var panels = new Dictionary<IngameInfoPanel, (string Panel, string Label, Action<ButtonWidget, Widget> Setup)>()
 			{
-				{ IngameInfoPanel.Objectives, ("OBJECTIVES_PANEL", "Objectives", SetupObjectivesPanel) },
-				{ IngameInfoPanel.Map, ("MAP_PANEL", "Briefing", SetupMapPanel) },
-				{ IngameInfoPanel.LobbbyOptions, ("LOBBY_OPTIONS_PANEL", "Options", SetupLobbyOptionsPanel) },
-				{ IngameInfoPanel.Debug, ("DEBUG_PANEL", "Debug", SetupDebugPanel) },
-				{ IngameInfoPanel.Chat, ("CHAT_PANEL", "Chat", SetupChatPanel) }
+				{ IngameInfoPanel.Objectives, ("OBJECTIVES_PANEL", modData.Translation.GetFormattedMessage(Objectives), SetupObjectivesPanel) },
```

Comment:
Likewise here. Have the consumer do the `GetFormattedMessage` instead of duplicating here.